src/batches/batchDetails.tsx

```diff
@@ -264,6 +273,15 @@ function BatchDetails({
           )}
         </div>
       )}
+        {createBatch && (
+          <CreateBatch
+          setCreateBatch={setCreateBatch}
+          regionName={regionName}
+          projectName={projectName}
+          batchInfoResponse={batchInfoResponse}  
+          createBatch= {createBatch}          />
+          )
```

Comment:
Fix the formatting here?  please use prettier if you don't have it installed:

https://www.npmjs.com/package/prettier 

it's already installed as a dev dependency.

Reply:
resolved